civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -32,7 +32,9 @@ function civihr_employee_portal_css_alter(&$css) {
 
     _remove_resources($css, $removeFiles);
   } else {
-    _remove_shoreditch_resources($css);
+    if (!strpos(url(current_path(), array('absolute' => TRUE)), 'onboarding-form')) {
```

Comment:
Please use the `[]` array syntax

Also `drupal_get_path_alias()` returns `onboarding-form` when I'm on the onboarding-form page. Maybe a simple 

```if (drupal_get_path_alias() === 'onboarding-form')```

would be cleaner?

Reply:
Thanks @mickadoo that's better. Will do!